backend/packages/Upgrade/src/api/controllers/validators/SegmentInputValidator.ts

```diff
@@ -25,7 +38,9 @@ export class SegmentInputValidator {
   public userIds: string[];
```

Comment:
Array validations on `userIds` is remaining.

Reply:
it was already there